src/pymor/vectorarrays/interfaces.py

```diff
@@ -825,9 +829,15 @@ def __hash__(self):
 
     def __repr__(self):
         return f'{self.__class__.__name__}({self.id})'
-
-
-def _create_random_values(shape, distribution, random_state, **kwargs):
+        
+        
+def _is_complex_dtype(dtype):
+    if isinstance(dtype, np.dtype):
+        dtype = dtype.type
+    return dtype in [np.complex, np.complex64, np.complex128]
```

Comment:
Maybe `np.issubdtype(dtype, np.complexfloating)` is better? ([`np.issubdtype` docs](https://docs.scipy.org/doc/numpy/reference/generated/numpy.issubdtype.html), [NumPy scalars docs](https://docs.scipy.org/doc/numpy/reference/arrays.scalars.html))